dist/src/conf/dogma.json

```diff
@@ -28,14 +28,19 @@
   },
   "webAppEnabled": true,
   "webAppTitle": null,
-  "mirroringEnabled": null,
+  "mirroringEnabled": true,
```

Comment:
Could you please explain the rationale behind this change?

Reply:
I see in the documentation:
![image](https://github.com/line/centraldogma/assets/11583408/4c168c1f-d1d8-4169-bfec-d7f685d9eddb)

Reply:
Oops, I didn't notice that. 😓 Thanks!